build.sh

```diff
@@ -100,3 +100,6 @@ fi
 # Build EOS
 cmake -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DCMAKE_CXX_COMPILER=${CXX_COMPILER} -DCMAKE_C_COMPILER=${C_COMPILER} -DWASM_LLVM_CONFIG=${WASM_LLVM_CONFIG} -DBINARYEN_BIN=${BINARYEN_BIN} -DOPENSSL_ROOT_DIR=${OPENSSL_ROOT_DIR} -DOPENSSL_LIBRARIES=${OPENSSL_LIBRARIES} ..
 make -j4
+
+# Install to the systems (in order to run eoscpp globally)
+# sudo make install 
```

Comment:
Hey, just wondering why there is no these two lines in `build.sh`, the issue https://github.com/EOSIO/eos/issues/911 still happened on my side after I cloned this repo with Mac OS. Shall we reopen that issue?

Reply:
Oh, sorry my fault, this PR wasn't merged